ui/src/containers/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
```

Comment:
Let's make sure to add more tests soon'ish.

Reply:
yes! Unit tests is planned 👍